hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/ScheduleCompactionActionExecutor.java

```diff
@@ -140,6 +140,17 @@ private Option<Pair<Integer, String>> getLatestDeltaCommitInfo() {
     return Option.empty();
   }
 
+  private Option<Pair<Integer, String>> getLatestDeltaCommitInfoSinceCompactionRequest() {
+    Option<Pair<HoodieTimeline, HoodieInstant>> deltaCommitsInfo =
```

Comment:
deltaCommitsInfo or deltaCommitInfo?

Reply:
I think using deltaCommitsInfo here should be ok since the left value is delta Commits after the right value, WDYT?